packages/sn-filter-pane/src/ext/conversion/list-object-util.ts

```diff
@@ -0,0 +1,41 @@
+import extend from 'extend';
+
+export default {
+  createDefaultListBox(dimensionDef, title) {
+    const listboxDef = extend(true, {}, dimensionDef);
+    let i;
+    listboxDef.qShowAlternatives = true;
+    // listboxDef.qDirectQuerySimplifiedView = directQueryAdaptService.adaptationsEnabled();
+    for (i = 0; i < listboxDef.qDef.qSortCriterias.length; ++i) {
+      listboxDef.qDef.qSortCriterias[i].qSortByState = 1;
+    }
+    if (!title && dimensionDef.qDef.qLabelExpression) {
+      title = {
```

Comment:
Hmm should probably create a new variable called e.g. `newTitle` (although its old code).